Jasmine/Grid/ViewControllers/GridGameViewController.swift

```diff
@@ -21,6 +21,10 @@ class GridGameViewController: UIViewController {
     /// Set its theme after the view controller `viewDidLoad` is called.
     override func viewDidLoad() {
         setTheme()
+
+        if viewModel.gameStatus == .notStarted {
```

Comment:
if you want to start game right after the screen loads, should place this at viewDidAppear. Did load is pretty early for this